extension/spark-doris-connector/src/main/java/org/apache/doris/spark/cfg/ConfigurationOptions.java

```diff
@@ -40,9 +40,11 @@
     String DORIS_REQUEST_RETRIES = "doris.request.retries";
     String DORIS_REQUEST_CONNECT_TIMEOUT_MS = "doris.request.connect.timeout.ms";
     String DORIS_REQUEST_READ_TIMEOUT_MS = "doris.request.read.timeout.ms";
+    String DORIS_REQUEST_QUERY_TIMEOUT_S = "doris.request.query.timeout.s";
     int DORIS_REQUEST_RETRIES_DEFAULT = 3;
     int DORIS_REQUEST_CONNECT_TIMEOUT_MS_DEFAULT = 30 * 1000;
     int DORIS_REQUEST_READ_TIMEOUT_MS_DEFAULT = 30 * 1000;
+    int DORIS_REQUEST_QUERY_TIMEOUT_S_DEFAULT = -1;
```

Comment:
I think we would better set a reasonable default value. one hour is enough for most cases.

Reply:
Maybe we could set other default value, but spark on doris would get a wrong answer if query time exceeds default value without any error or warning.  This wrong answer would mislead users.

Reply:
If query timeout, we should make this query fail to users.

Reply:
OK，the default value of query timeout would set be one hour, and this pr just allow users to define query timeout option, it would fix bug "query fail when timeout" in next pr, is that OK?

Reply:
That will be nice